backend/configuration/configure_appwrite.py

```diff
@@ -205,13 +205,14 @@ def set_up_collections(database: Database):
     """
     existing_collections = get_existing_collection_names(database)
     print(f"Already existing collections: {existing_collections}")
+    admins_id = get_or_create_team_id(teams, 'Admins')
 
     if "ABIs" not in existing_collections:
         print('Create "ABIs" Collection ...', end="")
         database.create_collection(
             "ABIs",  # Collection Name
             ["*"],  # Read permissions
-            ["team:Admins"],  # Write permissions
+            [f"team:{admins_id}"],  # Write permissions
```

Comment:
Did you test that this is really needed? We should not make superfluous commits.
Locally on my PC, it worked with only `Admins`, the team name is fine. But it maybe doesn't work for you if you team is not called `Admins`.

Reply:
now both versions are added - so that no potential incompatibles are accure